src/clj/rems/locales.clj

```diff
@@ -26,17 +28,42 @@
         translations-path (str path "/extra-translations/")]
     translations-path))
 
+(defn extract-format-parameters [string]
+  (when (string? string)
+    (set (re-seq #"%\d+" string))))
```

Comment:
If this would always return a set instead of nil, it would avoid the unnecessary warning about unused keys.

Reply:
Good point